lint-workflow/lint.py

```diff
@@ -190,10 +344,24 @@ def main():
                     input_files.append(filepath)
     else:
         print("File/Directory does not exist, exiting.")
+        return -1
+
+    max_error_level = 0
```

Comment:
Consider a more functional approach (lines 349 - 353):

```suggestion
    max_error_level = max(list(map(lint, input_files)))
```